src/include/miopen/fusion/problem_description.hpp

```diff
@@ -95,25 +95,26 @@ struct FusionDescription
 #endif
 
     // This and the following method should be moved to the Ops once the return type can be unified
-    miopen::ProblemDescription GetConvProblem(size_t idx, conv::Direction dir) const
+    miopen::ProblemDescription
+    GetConvProblem(size_t idx, conv::Direction dir, bool do_bias = false) const
     {
         const auto& conv_op =
             dynamic_cast<ConvForwardOpDescriptor&>(*fusion_plan_desc->op_map[idx]);
         if(dir == conv::Direction::Forward)
         {
             TensorDescriptor out_desc;
             conv_op.GetOutputDesc(out_desc);
-            return miopen::ProblemDescription{conv_op.input_desc,
-                                              conv_op.filter_desc,
-                                              out_desc,
-                                              conv_op.base_desc /* conv desc */,
-                                              dir};
+            return miopen::conv::ProblemDescription{conv_op.input_desc,
+                                                    conv_op.filter_desc,
+                                                    out_desc,
+                                                    conv_op.base_desc /* conv desc */,
+                                                    dir,
+                                                    do_bias ? 1 : 0};
```

Comment:
[R][clarity] Can we make `conv::ProblemDescription` to accept bool here?

Reply:
I guess it would be even better to remove `do_bias` and use `bias` instead of it.

Reply:
Up to you. Resolved.